khard/khard.py

```diff
@@ -1030,6 +1034,8 @@ def list_subcommand(vcard_list, parsable, fields):
     :type vcard_list: list of carddav_object.CarddavObject
     :param parsable: machine readable output: columns devided by tabulator (\t)
     :type parsable: bool
+    :param fields: list of strings for field evaluation
+    :type fields: list
```

Comment:
list(str)

Reply:
Is this used to generate some documentation somewhere?

Otherwise, this type of repetitious documentation is not helping anyone,
would make the writers of https://www.python.org/dev/peps/pep-0257 sad.

Please update [CONTRIBUTING.rst](https://github.com/scheibler/khard/blob/develop/CONTRIBUTING.rst)
if this type of documentatino needs to be written.

(otherwise, feel free to add it yourself).

Reply:
These are used in the sphinx generated documentation.  Try `make -c doc html && $BROWSER doc/build/html/index.html` locally or go to https://khard.readthedocs.io/ .  Although the python modules (and these signatures) are not yet visible on readthedocs (only after the next release).

I tend to put a more "human" description in the `:param:` part and the explicit type signature in the `:type:` part. 

```suggestion
    :param fields: the object path to the field on the vcard that should be printed
    :type fields: list(str)
```

But I can fix these small things when I merge. I know that I am very picky about such things.

About duplication info: I am also thinking about real python type signatures in #204. But that is another big project and seems to bring its own complications.